apps/api/src/server/applications/application/__tests__/get-application-details.test.js

```diff
@@ -12,6 +12,10 @@ const application1 = applicationFactoryForTests({
 	title: 'EN010003 - NI Case 3 Name',
 	description: 'EN010003 - NI Case 3 Name Description',
 	caseStatus: 'draft',
+	gridReference: {
+		easting: 123_456,
+		northing: 987_654
+	},
```

Comment:
It doesn't seem like this is used by the `applicationFactoryForTests` function, so we can probably remove this

Reply:
removed it